File: common-client-api-libs/src/main/java/eu/getsoftware/hotelico/clients/api/clients/domain/chat/IChatMessageView.java
package eu.getsoftware.hotelico.clients.api.clients.domain.chat;

import java.sql.Timestamp;

public interface IChatMessageView {
    boolean isActive();

    String getMessage();

    long getInitId();

    Timestamp getTimestamp();

    long getSenderId();

    boolean isSeenByReceiver();

    boolean isDelieveredToReceiver();

    long getReceiverId();

    String getSpecialChatContent();
}

File: shared-client-libs/src/main/java/eu/getsoftware/hotelico/clients/api/clients/domain/chat/IChatMessageView.kt
package eu.getsoftware.hotelico.clients.api.clients.domain.chat;

import java.sql.Timestamp;

public interface IChatMessageView {
    boolean isActive();

    String getMessage();

    long getInitId();

    Timestamp getTimestamp();

//    CustomerDomainEntityId getSenderId();

    boolean isSeenByReceiver();

    boolean isDelieveredToReceiver();

//    CustomerDomainEntityId getReceiverId();

    String getSpecialChatContent();
}
